DotNetExercise/Exercise05.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DotNetExercise
+{
+    public class Exercise05
+    {
+        public static string ExecuteQuadraticEquation(int number1, int number2, int number3)
+        {
+            string result = "";
+            double delta, x1, x2;
+            delta = number2 * number2 - 4 * number1 * number3;
+            if (delta > 0)
+            {
+                x1 = -number2 + Math.Sqrt(delta) / 2 * number1;
+                x2 = -number2 - Math.Sqrt(delta) / 2 * number1;
+                result += "x1 = " + x1 + ", " + "x2 = " + x2;
```

Comment:
```suggestion
                return "x1 = " + x1 + ", " + "x2 = " + x2;
```

Áp dụng tương tự cho 2 cái dưới kia

Reply:
Done